source/code/include/playfab/PlayFabPlatformUtils.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <chrono>
+#include <ctime>
+#include <iomanip>
+#include <sstream>
+
+namespace PlayFab
+{
+    typedef std::chrono::time_point<std::chrono::system_clock> TimePoint;
+
+    inline TimePoint GetTimePointNow()
+    {
+        return std::chrono::system_clock::now();
+    }
+
+    inline time_t GetPlayFabTimeTNow()
+    {
+        time_t now = std::chrono::system_clock::to_time_t(GetTimePointNow());
+#if defined(PLAYFAB_PLATFORM_IOS) || defined(PLAYFAB_PLATFORM_ANDROID) || defined(PLAYFAB_PLATFORM_LINUX) || defined(PLAYFAB_PLATFORM_SWITCH)
+        return static_cast<Json::Int64>(now);
+#else // PLAYFAB_PLATFORM_IOS || PLAYFAB_PLATFORM_ANDROID || PLAYFAB_PLATFORM_LINUX || PLAYFAB_PLATFORM_SWITCH
+        return now;
+#endif // PLAYFAB_PLATFORM_IOS || PLAYFAB_PLATFORM_ANDROID || PLAYFAB_PLATFORM_LINUX || PLAYFAB_PLATFORM_SWITCH
+    }
+
+    inline void AppendIntToString(int value, std::string& output)
```

Comment:
>AppendIntToString [](start = 16, length = 17)

In the earlier iteration of this change I suggested replacing this with calls to std::to_string. You suggested that this wasn't supported on all platforms. What platforms don't support this? It's a std library function. as long as you have c++11 support (which is required by the playfab API for other reasons) you should be fine.

[https://en.cppreference.com/w/cpp/string/basic_string/to_string](https://en.cppreference.com/w/cpp/string/basic_string/to_string)

Reply:
I commented that std::to_string doesn't exist in all environments.

Reply:
what environments/platforms? I'm arguing that it has to because it's part of the std library that you already depend on and you haven't seemed to provide a counter example that disproves that

Reply:
I can't argue with your logic. It's possible that's been resolved. But we don't have automated testing across everything right now, so I'm trying to limit myself to the safest and most relevant changes. This function exists, and it has been tested to work on all platforms. std::to_string definitely didn't exist on at least one platform in the past, which forced us to make this function.
Out of scope for this PR.
Task filed.